utils/configuration_test.go

```diff
@@ -178,34 +178,22 @@ func TestParseInvalidStorageNoDBSource(t *testing.T) {
 		`{"storage": {"backend": "%s"}}`,
 		`{"storage": {"backend": "%s", "db_url": ""}}`,
 	}
-	for _, backend := range []string{MySQLBackend, SqliteBackend} {
+	for _, backend := range []string{notary.MySQLBackend, notary.SQLiteBackend} {
 		for _, configJSONFmt := range invalids {
 			configJSON := fmt.Sprintf(configJSONFmt, backend)
-			_, err := ParseStorage(configure(configJSON),
-				[]string{MySQLBackend, SqliteBackend})
+			_, err := ParseSQLStorage(configure(configJSON))
 			require.Error(t, err, fmt.Sprintf("'%s' should be an error", configJSON))
 			require.Contains(t, err.Error(),
 				fmt.Sprintf("must provide a non-empty database source for %s", backend))
 		}
 	}
 }
 
-// If a memory storage backend is specified, no DB URL is necessary for a
-// successful storage parse.
-func TestParseStorageMemoryStore(t *testing.T) {
```

Comment:
Since we're parsing memory stores in the `cmd/server` and `cmd/signer` packages now, it'd be nice to move this test to each of them (e.g. that we don't need any additional params for the memory store).